mw-migrations/src/migrations/deleteFavoritesInWaysAndUsers.ts

```diff
@@ -0,0 +1,89 @@
+import { doc, writeBatch } from "firebase/firestore";
+import { db } from "../firebase.js";
+import { PATH_TO_USERS_COLLECTION, UsersService } from "../service/UsersService.js";
+import { PATH_TO_WAYS_COLLECTION, WayService } from "../service/WayService.js";
+import { logToFile } from "../utils/logToFile.js";
```

Comment:
I think better to create 2 separate migration files for different models. Also please rename file similar as files in migration folder

Reply:
Done.